lib/pubsuffix-psl.ts

```diff
@@ -84,5 +84,5 @@ export function getPublicSuffix(
     )
   }
 
-  return psl.get(domain)
+  return getDomain(domain)
```

Comment:
```suggestion
  return getDomain(domain, { allowPrivateDomains: true })
```

I think we should follow through with @remusao's suggestion. :)